INTERESTED_DEVELOPERS.md

```diff
@@ -0,0 +1,25 @@
+# Interested Developers:
+This file will help us track people that are interested in taking decisions of the specification of GraphQL over HTTP until migrated to a official repository of the https://github.com/graphql organization.
+
+If you want to be listed here, open a PR with your information, just order yourself by name.
+
+## List of Developers:
+* @deinok
+  * PL: .NET
+  * Framework: dotnet-graphql
+  * Interested in: BOTH
+  * Repos:
+    * https://github.com/graphql-dotnet/graphql-client
```

Comment:
I think it could be discouraging for a person who doesn't have popular GitHub repo.
How about `Company/Project`?
So a person can represent his company or his project or both.

\+ I'm not sure we need all these details, I would suggest to shorten it to 
GitHub nick and `Company/Project`.

Reply:
Well, I would like to let the repos, I think that the most important thing of this document is to have somebody representating every framework (Apollo, graphql.js, graphql-dotnet, graphql-java, etc...) so all the main developers of this frameworks can accept to support this spec.

As an example, I represent the main developer of the client for .NET but, I'm not the main developer of the .NET Http Server.

This is just for track the people that want to be informed of the progress.
I mean, the spec itself its open to discuss to anybody.

Reply:
I'm mean repo is a just a particular form of a project so e.g. `graphql-js`, `graphql-client` and others are projects.
To prevent confusion it can be `Company/Project/Repo`.

> so all the main developers of this frameworks can accept to support this spec.

This is the idea behind this effort. My comment was about the different situation, e.g. couple months ago I spoke with a developer from `AWS AppSync` about contributing to this spec, so he could represent `AWS AppSync` which AFAIK doesn't have public GitHub repo and don't represent particular language or framework.
The similar story, with other BaSS like Hasure, Prisma, etc.
My personal motivation for this spec is to solve the interoperability problem for our tools like Voyager, Faker, etc. So I'm also don't represent any language or framework.

> As an example, I represent the main developer of the client for .NET but, I'm not the main developer of the .NET Http Server.

So it could be:
```
@deinok:
  Company/Project/Repo: https://github.com/graphql-dotnet/graphql-client
  Notes: Interested in client/server on C# stack
```
If you think it's important to share something just add it to the `Notes`.
Does it make sense?

Reply:
Yes, I didn't think about this case